### ISSUE TITLE
Web client or web interface

### ISSUE THREAD

**Issue by narcisgarcia** [gobby/gobby#9]
A way to access to Gobby service via web browser should improve the easy of an instant meeting and writing.
http://gobby.0x539.de/trac/ticket/587

**Comment by bojidar-bg** [comment]
:+1: if there is a javascript library, it should be easy to integrate with Atom. :smile:

**Comment by NK-Analyst** [comment]
Greetings one and all.

I have time to do something helpful. Could you tell me, is web client or web interface is needed at the present time?
If "Yes". How can I help implement the web client or web interface? 

Thank you.

**Comment by valessiobrito** [comment]
I don't understand programming complexity. But you may need to connect the Gobby protocol with a web application to write/read notes. Looking at the history, maybe connecting these two things (jinfinote+litewrite) to a product for "GobbyWeb".

https://github.com/sveith/jinfinote + http://litewrite.net = "GobbyWeb"

**Comment by NK-Analyst** [comment]
Thank you for answer.

I will analyze jinfinote and litewrite.net. At the present time I have a plan in my head.
1) I will find out the requirements for the "GobbyWeb". Speaking of requirements I mean User Stories and Use Cases.
2) I will prepare mockups.
3) I will implement working prototype "GobbyWeb".
4) For users, I will present the results of each steps above to get feedback.

Could You tell me, will you have the opportunity to give me feedback and could you help me in finding users for feedback?

What do you think of my plan?

Thank you.

**Comment by valessiobrito** [comment]
Yes, I can collaborate with feedback. But I think the idea of a "GobbyWeb" is to replicate the basic and essential features of Gobby Desktop. That is, as a user, indicate a Gobby server to browse notes that are saving or create new notes. Anyway, I believe it is important to see the Wiki, it seems that already had a documented work https://github.com/gobby/gobby/wiki/Development

**Comment by valessiobrito** [comment]
Another idea/alternative is to create an extension for [NextCloud](https://github.com/nextcloud) to support the Gobby protocol. I just opened an Issue over https://github.com/nextcloud/server/issues/15400

**Comment by NK-Analyst** [comment]
Thank You :)

I went to analyze jinfinote, litewrite.net, wiki and NextCloud.

Have a good one.

**Comment by NK-Analyst** [comment]
Greetings one and all.

@valessiobrito , I have analyzed information about jinfinote, litewrite and Gobby. After it in my head arised some ideas about implement GobbyWeb.
I started experimenting. When I shall get any interesting results, I shall report about it.

Thank you for the help.

**Comment by narcisgarcia** [comment]
Thank you all for working about this.
A web alternative for Gobby could relaunch project popularity.

**Comment by NK-Analyst** [comment]
Greetings one and all.
I just report. I continue to work on the project. I need to study GLib (https://developer.gnome.org/glib/) to better understand some things in libinfinity. It will take some time. I will show the results in September.

**Comment by NK-Analyst** [comment]
Greetings one and all.

I had stopped work on web-client for infinoted at 28 september :( 
Based on the results of myself research, I came to the conclusion that it work is too big for one person. At present time I do not have enough resources to form dev team.

I thanks gobby and infinoted teams for their good work. You are great.

**Comment by narcisgarcia** [comment]
NK-Analyst, If you want server/web resources for this project, please contact me via private message.
If Github doesn't allow user-to-user messages, let me know here.

**Comment by NK-Analyst** [comment]
@narcisgarcia , thanks for help :)

Speaking about resources i meant the following.

1. Time for building strategy of software development.
2. A strong idea that will interest experienced developers.

Now I want to relax a bit (the research took a lot of my strength). Perhaps during the rest I will have new ideas about this project.

At present time I have come to the conclusion that for the web-client to work we should be realize some middleware. 
Middleware should be between the web-client (html + css + js) and the infinoted.
The core of middleware must be written in C to interact with infinoted. The current state of affairs does not allow the use of higher-level programming languages.

**Comment by NK-Analyst** [comment]
I did not publish my work on Github because it is very raw and will confuse those people who might want to make a web-client in the future.

**Comment by narcisgarcia** [comment]
To not losing your fresh research information, could you document your results in a Wiki or similar? This could be useful to continue (and not wasting time into same research again) a next interested developer.

**Comment by NK-Analyst** [comment]
Of course, of course. A little later I will prepare a wiki document.

**Comment by NK-Analyst** [comment]
@narcisgarcia , I can't send private message to you via GitHub. Could you tell me please how to contact with you via any another way.

**Comment by narcisgarcia** [comment]
Please make a first contact to: nogobby AT actiu.net
